homeassistant/components/freebox/router.py

```diff
@@ -146,6 +155,28 @@ async def _update_disks_sensors(self) -> None:
         for fbx_disk in fbx_disks:
             self.disks[fbx_disk["id"]] = fbx_disk
 
+    async def update_home_devices(self) -> None:
+        """Update Home devices (light, cover, alarm, sensors ...)."""
+        new_device = False
+        # Home sensors (alarm, pir, switch, remote ...)
+        try:
+            home_nodes: list[Any] = await self.home.get_home_nodes() or []
+        except InsufficientPermissionsError:
```

Comment:
I don't think this catch is enough as I got this error on my side (Freebox Revolution):
```shell
2023-04-25 11:39:28.028 ERROR (MainThread) [homeassistant.config_entries] Error setting up entry REDACTED.fbxos.fr for freebox
Traceback (most recent call last):
  File "/workspaces/core/homeassistant/config_entries.py", line 383, in async_setup
    result = await component.async_setup_entry(hass, self)
  File "/workspaces/core/homeassistant/components/freebox/__init__.py", line 60, in async_setup_entry
    await router.update_all()
  File "/workspaces/core/homeassistant/components/freebox/router.py", line 86, in update_all
    await self.update_home_devices()
  File "/workspaces/core/homeassistant/components/freebox/router.py", line 163, in update_home_devices
    home_nodes: list[Any] = await self.home.get_home_nodes() or []
  File "/home/vscode/.local/lib/python3.10/site-packages/freebox_api/api/home.py", line 170, in get_home_nodes
    return await self._access.get("home/nodes")
  File "/home/vscode/.local/lib/python3.10/site-packages/freebox_api/access.py", line 122, in get
    return await self._perform_request(self.session.get, end_url)
  File "/home/vscode/.local/lib/python3.10/site-packages/freebox_api/access.py", line 114, in _perform_request
    raise HttpRequestError(err_msg)
freebox_api.exceptions.HttpRequestError: Request failed (APIResponse: {"msg": "Version d'API inconnue", "success": false, "error_code": "invalid_api_version"})
```

Using `InsufficientPermissionsError` parent `HttpRequestError` fixes the issue.

```suggestion
        except HttpRequestError:
```